andela/travel_tool_front
# #161502437 Fix add new member modal

#### What does this PR do?
This PR fixes the `Add user modal` issue which occurs immediately after opening the `edit modal`  on the `travel team page`.

#### Description of Task to be completed?
- Update the `openModal` method for the `add user` button click
- Update tests to suit implementation

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- Install dependencies and start the  app - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/139)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login
- Go to the`DB` and on the `UserRoles` table, set yourself as a `Travel Admin`
- Go to the browser, refresh the page, Navigate to the `Settings` tab and  click on `Travel Team`
- Click `edit` on any travel team member available on the page.
- Close the edit modal and immediately click the `add user` button
- The modal should display `Add user form`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161502437](https://www.pivotaltracker.com/story/show/161502437)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
